### PR TITLE
feat: CameraPickerConfig 新增 FlashMode，设置相机默认闪光灯模式。

### PR DESCRIPTION
**Pull request by leftcoding**
CameraPickerConfig 新增 FlashMode，设置相机默认闪光灯模式。

### CODE REVIEW

File: lib/src/states/camera_picker_state.dart [hunk 1]
```diff
@@ -881,9 +863,6 @@ class CameraPickerState extends State<CameraPicker>
         icon = Icons.flashlight_on;
         break;
     }
-    if (shouldSwitchFlashMode) {
```
Comment (AlexV525) [inline lib/src/states/camera_picker_state.dart]: Why we need to remove this?

File: lib/src/constants/config.dart [hunk 2]
```diff
@@ -138,6 +139,10 @@ class CameraPickerConfig {
   /// 摄像机在拍摄时锁定的旋转角度
   final DeviceOrientation? lockCaptureOrientation;
 
+  /// The possible flash modes that can be set for a camera.
+  /// 可以为相机设置闪光模式
+  final FlashMode flashMode;
```
Comment (AlexV525) [inline lib/src/constants/config.dart:144]: Please reference to `preferredLensDirection` and apply the similar comments.

```suggestion
  final FlashMode preferredFlashMode ;
```

File: lib/src/states/camera_picker_state.dart [hunk 3]
```diff
@@ -329,6 +329,7 @@ class CameraPickerState extends State<CameraPicker>
             newController
                 .getMinZoomLevel()
                 .then((double value) => minAvailableZoom = value),
+            newController.setFlashMode(pickerConfig.flashMode),
```
Comment (AlexV525) [inline lib/src/states/camera_picker_state.dart:332]: Set flash mode only when it's not `.auto`?

File: lib/src/constants/config.dart (hunk 2 above)
Comment (leftcoding) [reply lib/src/constants/config.dart:144]: get

**Pull request closed without merging**